libsubid/subid.h

```diff
@@ -13,5 +14,73 @@ enum subid_type {
 	ID_TYPE_GID = 2
 };
 
+enum subid_status {
+	SUBID_STATUS_SUCCESS = 0,
+	SUBID_STATUS_UNKNOWN_USER = 1,
+	SUBID_STATUS_ERROR_CONN = 2,
+	SUBID_STATUS_ERROR = 3,
+};
+
+struct subid_nss_ops {
```

Comment:
What is the reason to keep this definition in `libsubid/*`?
It seems it is only used in `lib/nss.c` and `lib/subordinateio.c`.

Reply:
Seemed the thing to do at the time.

I've moved it to lib/prototypes.h.